from llaim.exception import LLAIMException


class LLAIMEtlException(LLAIMException):
    pass
